Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM registry.fedoraproject.org/fedora:31
+
+WORKDIR /tmp/mantle
+COPY . /tmp/mantle
+
+RUN yum update -y --nodocs && \
+    yum install -y --nodocs make golang gcc && \
+    make && \
+    make install && \
```

Comment:
If using multi-stage this would be `make install DESTDIR=/tmp/mantle` and then `COPY --from=builder /tmp/mantle/usr/bin/* /usr/bin` or so.

Reply:
Good point, will update!